lib/cli.mjs

```diff
@@ -238,7 +238,7 @@ const flags = {
 		type: 'boolean',
 	},
 	fix: {
-		type: 'boolean',
+		type: 'string',
```

Comment:
As explained [here](https://github.com/stylelint/stylelint/issues/7497#issuecomment-2085529108), you don't have to change the type.
i.e. leave `boolean` and retrieve the value in the `if`

Reply:
I see your point, but shouldn't it be treated as a `string` rather than a `boolean`? Definig it as a `boolean` and then retrieving a `string` value seems inconsistent and relies on an upstream issue.
I'm following the approach shown here.
https://github.com/noahbrenner/xo/blob/8e4f4355c4b8e62bc28f37e3673bb6872e693bb0/main.js#L119-L133

Reply:
Well either is fine but if we leave it as `'boolean'`, it will reduce the diff considerably.
e.g. the default remains `false`

Reply:
I'm not 100% sure, but can `boolean` accept `--fix=lax`?

IMO, if the code doesn't become more complex, `string` seems more acceptable to me.

However, the change should be limited within `cli.mjs`, so either seems to have no big problems.